src/pages/Career/CareerPart1.js

```diff
@@ -0,0 +1,43 @@
+import React, { useRef } from "react";
+import Scrolling from "./Scrolling";
+
+
+
+const CareerPart1 = () => {
+    const ref = useRef(null)
+    const handleClick = () => {
+        ref.current?.scrollIntoView({behavior: 'smooth'})
```

Comment:
Great job using scrollIntoView! I love it!

Should we also show/hide current open positions  list by making this function toggle a useState with true/false values and use a ternary in order to display the second part?